src/lib/data/near.js

```diff
@@ -111,6 +166,13 @@ async function accountState(near, accountId) {
 }
 
 async function sendTransactions(near, functionCalls) {
+  if (functionCalls.length == 1) {
+    const { contractName, methodName, args, gas, deposit } = functionCalls[0];
```

Comment:
I can't imagine zero functionCalls here, but nothing is changed on how this is handled. Also here, there is a check if the number of functionCalls are 1. Otherwise the previous behaviour with confirmation through the wallet will apply

Reply:
I was probably sleepy at that moment and somehow missed that